### DISCUSSION TITLE
A gang of Gweazles

### DISCUSSION THREAD

**Discussion by ph0racious** [keirf/greaseweazle#335] [Q&A] [answered]
Curious if the existing builds or any forks support, or plan to support, multiple Greaseweazles attached to the same host?

I have a large archiving project in the works, setting up a 'jukebox' of 10 Gweazles that have just been ordered, offhanded thoughts are to just containerize (docker) the setup but if the software could do it natively it'd be far easier...

**Comment by keirf** [comment, marked as answer]
The gw command line program can be told which Greaseweazle device to communicate with, via the /dev path (Linux and Mac) or COMx (Windows). This has the disadvantage that these paths might change. I could add identification by serial but the Gweazle serial strings are long and unmemorable.

On Linux, since I suspect you use Linux, you could have udev assign memorable names to your gweazles under /dev. You could use the existing udev script as a starting point, because it already creates a /dev/greaseweazle symlink. This could be extended to match on serial and create a more specific name for each device.

**Reply by ph0racious** [reply]
Thank you!
